fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -647,7 +677,14 @@ LSTATUS __stdcall RegOpenKeyTransactedFixup(
     Log("[%d] RegOpenKeyTransacted:\n", RegLocalInstance);
 #endif
 
+    const char* updatedSubKey = ReplaceRegistryQueryPath(&key, subKey);
+    if (updatedSubKey)
```

Comment:
Looks like line 596 and 680 are the only places where `ReplaceRegistryQueryPath` is called and, in both instances `key` is not used.  Maybe change the arguments of `ReplaceRegistryQueryPath to HKEY instead of PHKEY.`

Reply:
Updated `key `is used in the next registry call at line 599 and 683

Reply:
I see.  Sorry.  Why does `ReplaceRegistryQueryPath` change key to HKCU?